src/components/CameraPermissions/Primer/index.tsx

```diff
@@ -70,4 +70,4 @@ const Permissions: FunctionComponent<Props> = ({
   )
 }
 
-export default trackComponent(localised(Permissions))
+export default trackComponent(localised(Permissions), 'camera_access')
```

Comment:
There are more flows that use this, you will need to add all those event to the code.

Missing:
`screen_document_document_video_capture_camera_acces`
`screen_face_face_video_capture_camera_access`